docs/design/api/storage.rst

```diff
@@ -0,0 +1,324 @@
+Storage API
+###########
+
+For this examples let's say:
+
+- Buyers logs with Firefox Account Bearer Tokens
+- Seller apps logs with Hawk
+- Payment App logs with Basic Auth
+
+
+As a user, I want to retrieve my information about my purchases
+===============================================================
+
+.. code-block:: http
+
+    GET /buckets/mozilla/collections/payments/records
+    Authorization: Bearer <User Firefox Account Bearer Token>
```

Comment:
We've been using http://pythonhosted.org/sphinxcontrib-httpdomain/ for our docs, there's still a lot of typing, but this would look like:

'''
.. http:get:: /buckets/mozilla/collections/payments/records

  **Request**
'''